### PR TITLE
translation for QA

### PR DESCRIPTION
**Pull request by lisa0314**
@jondong , please help to review

### CODE REVIEW

File: public/documentation/qa/quality_dashboard_zh.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Quality Dashboard
+下表包含了按照组件和特性组织的每个版本的测试结果。详细的测试报告链接包含在没有通过100%测试样例的特性中。
+
+## 通过组件
```
Comment (jondong) [inline public/documentation/qa/quality_dashboard_zh.md]: 按组件分类

File: public/documentation/qa/quality_dashboard_zh.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+<quality test-type="components"></quality>
+
+## 通过特性
```
Comment (jondong) [inline public/documentation/qa/quality_dashboard_zh.md]: 按特性分类

**Comment by jondong** [comment]
@lisa0314 review done, thanks for the PR

**Comment by lisa0314** [comment]
@jondong , thanks for review. All suggestions done.

**Pull request merged**